projects/mercury/src/collections/CollectionInformationDrawer.js

```diff
@@ -231,15 +228,17 @@ const PathMetadata = React.forwardRef(({path, showDeleted, hasEditRight = false,
 
     let body;
     let isDirectory;
+    let cardTitle = "Metadata";
     let avatar = <FolderOpenOutlined />;
     if (error) {
         body = <MessageDisplay message="An error occurred while determining metadata subject" />;
     } else if (loading) {
         body = <div>Loading...</div>;
     } else if (!data || !data.iri) {
-        body = <div>No metadata found</div>;
+        body = <div>Select an item to view it&apos;s metadata</div>;
```

Comment:
This one should be shown probably only when the path is empty, not on all empty data responses (`!data || !data.iri`) ?

Reply:
Directories always have metadata (unless the system is corrupt), so in practice if data is undefined it means no items are selected yet.